src/App.tsx

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [total] = useState(42);
```

Comment:
you don't change it
no need to save such values in the state
only values that change should be stored in the state (and this change should cause re-render)